plugin/src/Config.lua

```diff
@@ -1,6 +1,7 @@
 local strict = require(script.Parent.strict)
 
-local isDevBuild = script.Parent.Parent:FindFirstChild("ROJO_DEV_BUILD") ~= nil
+-- local isDevBuild = script.Parent.Parent:FindFirstChild("ROJO_DEV_BUILD") ~= nil
+local isDevBuild = true
```

Comment:
I recommend testing in a place where you've run `_G.ROJO_DEV_CREATE()` instead of hard-coding this constant one way or the other.